libs/ui-watt/src/lib/components/input/date-range-input/watt-date-range-input.component.ts

```diff
@@ -92,30 +127,174 @@ export class WattDateRangeInputComponent
   /**
    * @ignore
    */
-  placeholder: string = this.getPlaceholder(this.inputFormat);
+  @Input()
+  get placeholder(): string {
+    return this._placeholder;
+  }
 
   /**
    * @ignore
    */
-  initialValue?: WattDateRange | null = null;
+  set placeholder(value: string) {
+    this._placeholder = value;
+    this.stateChanges.next();
+  }
 
   /**
    * @ignore
    */
-  private destroy$: Subject<void> = new Subject();
+  private _placeholder: string = this.getPlaceholder(this.inputFormat);
+
+  /**
+   * @ignore
+   */
+  @Input()
+  get required(): boolean {
+    return this._required;
+  }
+
+  /**
+   * @ignore
+   */
+  set required(value: BooleanInput) {
+    this._required = coerceBooleanProperty(value);
+    this.stateChanges.next();
+  }
+
+  /**
+   * @ignore
+   */
+  private _required = false;
+
+  /**
+   * @ignore
+   */
+  @Input()
+  get disabled(): boolean {
+    return this._disabled;
+  }
+
+  /**
+   * @ignore
+   */
+  set disabled(value: BooleanInput) {
+    this._disabled = coerceBooleanProperty(value);
+    this.stateChanges.next();
+  }
 
   /**
    * @ignore
    */
+  private _disabled = false;
```

Comment:
issue: disabling the component doesn't really disable it. In the template, the `isDisabled` property is used, which is not toggled anymore. Here we're setting the `disabled` property. These two should be consolidated so there's only one property that tracks the disabled state and is used in the template.